app/forms/hyrax/pdf_form_behavior.rb

```diff
@@ -5,7 +5,14 @@ module PdfFormBehavior
     extend ActiveSupport::Concern
 
     included do
+      class_attribute :hidden_terms
+
       self.terms += %i[show_pdf_viewer show_pdf_download_button]
+      self.hidden_terms = %i[show_pdf_viewer show_pdf_download_button]
```

Comment:
I like your solution and think it'd be a helpful one to contribute back to hyku even, but I'm going to see if i can figure out why I can't just not include it in the form's terms.